robot/basestation/templates/pages/Base.html

```diff
@@ -35,6 +35,7 @@
     <!-- homemade -->
     <script type="text/javascript" src="../../static/js/env.js"></script>
     <script type="text/javascript" src="../../static/js/roslibjs/ros-helpers.js"></script>
+    <script type="text/javascript" src="../../static/js/roslibjs/Commands.js"></script>
```

Comment:
I think you should keep this in ros-helpers.js since these functions are literally ros utilities for our project.

Reply:
Fixed